.travis.yml

```diff
@@ -3,7 +3,9 @@ language: ruby
 sudo: false
 
 rvm:
-  - 2.2.2
```

Comment:
Can we keep support to 2.2?

Reply:
Sure, I added the needed pinnings to the Gemfile.